gammapy/background/energy_offset_array.py

```diff
@@ -104,3 +127,73 @@ def plot_image(self, ax=None, offset=None, energy=None, **kwargs):
         image = ax.imshow(self.data, extent=extent, **kwargs)
         plt.colorbar(image)
         return ax
+
+    @classmethod
+    def read(cls, filename):
+        """Create `EnergyOffsetArray` from  fits file.
+
+        Parameters
+        ----------
+        filename : str
+            File name
+        """
+        hdu_list = fits.open(filename)
```

Comment:
It's better to pass only the HDU, not the HDU list here:
```
hdu = hdu_list[1]
return cls.from_fits(hdu_list)
```
Then you have to remove the `[1]` in the `from_fits` method.

This is also what your docstring says, that `from_fits` is taking an HDU, not a HDUList as input.

OK?